static/title-bar.less

```diff
@@ -0,0 +1,35 @@
+@import "./variables/ui-variables";
+
+.title-bar {
+  height: 23px;
+
+  flex-shrink: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  font-family: -apple-system, BlinkMacSystemFont, "Helvetica Neue";
+  font-size: @title-bar-text-size;
+  color: @title-bar-text-focused;
+  -webkit-user-select: none;
```

Comment:
I think it would be good to already add `-webkit-app-region: drag;`. See http://electron.atom.io/docs/api/frameless-window/#text-selection

Not 100% sure, but otherwise dragging might stop working once that dragging fix lands in Atom.